packages/vscode-extension/package.nls.json

```diff
@@ -278,12 +278,13 @@
   "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.",
   "teamstoolkit.localDebug.learnMore": "Get More Info",
   "teamstoolkit.localDebug.m365TenantHintMessage": "After enrolling your developer tenant in Office 365 Target Release, enrollment may come into effect in couple of days. Click 'Get More Info' button for details on setting up dev environment to extend Teams apps across Microsoft 365.",
-  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps, you need to install both GitHub Copilot and GitHub Copilot Extension for Teams Toolkit (@teamsapp).",
-  "_teamstoolkit.handlers.askInstallCopilot.comment": "@teamsapp is used to call Teams agent in Github Copilot Chat and should not be translated.",
+  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, you need to install GitHub Copilot first.",
   "teamstoolkit.handlers.askInstallCopilot.install": "Install GitHub Copilot",
-  "teamstoolkit.handlers.askInstallCopilot.installTeamsApp": "Install @teamsapp",
-  "_teamstoolkit.handlers.askInstallCopilot.installTeamsApp.comment": "@teamsapp is used to call Teams agent in Github Copilot Chat and should not be translated.",
-  "teamstoolkit.handlers.installExtension.output": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps, you need to install both GitHub Copilot Chat following \"%s\" and GitHub Copilot Extension for Teams Toolkit (@teamsapp) following \"%s\".",
+  "teamstoolkit.handlers.askInstallTeamsAgent": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, you need to install GitHub Copilot Extension for Teams Toolkit. If you've already installed it, confirm.",
+  "teamstoolkit.handlers.askInstallTeamsAgent.install": "Install GitHub Copilot Extension",
+  "teamstoolkit.handlers.askInstallTeamsAgent.confirmInstall": "Confirm Installation",
+  "teamstoolkit.handlers.installCopilotAndAgent.output": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, you need to install GitHub Copilot following \"%s\" and GitHub Copilot Extension for Teams Toolkit from \"%s\".",
```

Comment:
Proposed content: 
To use the GitHub Copilot Extension for Teams Toolkit, install GitHub Copilot from \“%s\” and Github Copilot Extension from Teams Toolkit from “%s.”

Reply:
I think " when developing Teams apps or customizing Microsoft 365 Copilot" is needed as what Yan proposed before. This message will be shown in the output. How about 
To use the GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, install GitHub Copilot from \“%s\” and Github Copilot Extension from Teams Toolkit from “%s.”